### DISCUSSION TITLE
i want install php 8

### DISCUSSION THREAD

**Discussion by 0xezady** [Homebrew/discussions#3552] [Tap maintenance and Homebrew development]
i want install php 8 but i get this error:

curl: (22) The requested URL returned error: 403
Error: php@8.0: Failed to download resource "webp"
Download failed: https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.2.4.tar.gz

**Comment by carlocab** [comment]
That's an issue with your connection. Do `brew fetch webp` and try again. If that doesn't work, try using a different network.

**Reply by 0xezady** [reply]
it did not work

**Reply by gromgit** [reply]
Did you try using a different network? Because HTTP status code 403 (Forbidden) strongly suggests something is blocking you from downloading the above URL.

If you're behind the Great Firewall of China, or something similar, you'll have to ask your compatriots to help you resolve this.

**Reply by carlocab** [reply]
It works for me:
```
❯ brew fetch --build-from-source webp
==> Downloading https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.2.4.tar.gz
######################################################################## 100.0%
Downloaded to: /Users/carlocab/Library/Caches/Homebrew/downloads/d9612a63a401c09d477ac5095901cac903332c3762e2f8ff3f20da253e44bbb8--libwebp-1.2.4.tar.gz
SHA256: 7bf5a8a28cc69bcfa8cb214f2c3095703c6b73ac5fba4d5480c205331d9494df
```
There's an issue with your connection. You can try looking at the output of
```
curl -LI https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.2.4.tar.gz
```
or
```
curl -LIv https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.2.4.tar.gz
```
to try to see what the problem might be.
